system/file_windows.go

```diff
@@ -0,0 +1,15 @@
+// +build windows
+
+package system
+
+func (f *DefFile) Mode() (string, error) {
+	return "0000", nil // TODO implement
+}
+
+func (f *DefFile) Owner() (string, error) {
+	return getUserForUid(1000)
```

Comment:
no data is better than wrong data? What does the hardcoded 1000 do here?

Reply:
I'd rather return a blank or not-implemented.

Reply:
Done in 143e895